### PR TITLE
Filter out one switch flows from affected flows

### PR DESCRIPTION
**Pull request by timofei-durakov**
One switch flows should not be in a list of affected flows

### CODE REVIEW

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteV2Spec.groovy [hunk 1]
```diff
@@ -166,6 +190,7 @@ class AutoRerouteV2Spec extends HealthCheckSpecification {
         cleanup: "Remove the flow"
         flowHelperV2.deleteFlow(flow.flowId)
         isSwitchDisconnected && switchHelper.reviveSwitch(sw, blockData, true)
+        otherIslFailed && antiflap.portUp(otherIsl.srcSwitch.dpId, otherIsl.srcPort)
```
Comment (andriidovhan) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteV2Spec.groovy:193]: we should check that ISL is discovered
```
if (otherIslFailed) {
       antiflap.portUp(otherIsl.srcSwitch.dpId, otherIsl.srcPort)
        Wrappers.wait(discoveryInterval + WAIT_OFFSET) {
            assert northbound.getLink(otherIsl).actualState == IslChangeType.DISCOVERED
        }
}
```

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteV2Spec.groovy [hunk 2]
```diff
@@ -151,6 +152,29 @@ class AutoRerouteV2Spec extends HealthCheckSpecification {
             assert flowInfo.statusInfo == "Switch $sw.dpId is inactive"
         }
 
+        when: "Other isl fails"
+        def otherIslFailed = false
+        def otherIsl = topology.isls.find() {isl-> isl.srcSwitch != sw && isl.dstSwitch != sw}
```
Comment (andriidovhan) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteV2Spec.groovy:157]: as for me `otherIsl` is not clear
do you mind renaming it (for example to `islToFail`) ?

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteV2Spec.groovy [hunk 3]
```diff
@@ -151,6 +152,29 @@ class AutoRerouteV2Spec extends HealthCheckSpecification {
...
+        antiflap.portDown(otherIsl.srcSwitch.dpId, otherIsl.srcPort)
+        otherIslFailed = true
+        Wrappers.wait(WAIT_OFFSET) {
+            assert northbound.getLink(otherIsl).actualState == IslChangeType.FAILED
```
Comment (andriidovhan) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteV2Spec.groovy:161]: Why do you check the `actualState` field?
probably would be better to check the `state` field

**Comment by timofei-durakov** [comment]
> Log one-switch-flow's status changes via flows-dashboard-logger.

Added

**Review by surabujin — APPROVED** [review]

**Pull request merged**